### DISCUSSION TITLE
Can't upload image to Imgur, it is written that "server replied: Too Many Requests"

### DISCUSSION THREAD

**Discussion by trofey** [flameshot-org/flameshot#3590] [Q&A] [answered]
such issues appeared few month ago, but after some time everything worked fine, but this time there is no fix for maybe two month.

**Comment by mmahmoudian** [comment, marked as answer]
Please follow instructions here to create your own account in Imgur. You and other users are using our account which means our account goes out of quota fast. By creating your own account, you will not face the limitation of our quota.

https://flameshot.org/docs/guide/imgur-help/#creating-your-own-custom-client-id
